### DISCUSSION TITLE
mini.starter startify_skiplist like behaviour?

### DISCUSSION THREAD

**Discussion by ahmedelgabri** [nvim-mini/mini.nvim#776] [Q&A] [answered]
How to achieve the same result of ignoring specific files/paths from showing up in the list of recently viewed files? I guess that I will have to write my own `starter.sections.recent_files` to do this?

**Comment by echasnovski** [comment, marked as answer]
Yes, I am afraid so. The `recent_files` built-in section has only the [following arguments](https://github.com/echasnovski/mini.nvim/blob/a0edfb3a16533fb7335295f4554d81408301599d/doc/mini-starter.txt#L366-L371).

Another approach would be to ensure that certain files are not saved (or present at the time of `recent_files` is executed) in the `v:oldfiles` variable.
